### PR TITLE
add Astro

### PR DESCRIPTION
**Pull request by Ting-y**
Added astro framework to the files

### CODE REVIEW

File: files/astro/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "astro-wp",
+  "repo": "Ting-y/Astro",
+  "files": {
+    "include": ["astro-wp-element.min.js", "astro-support-views.min.js"]
```
Comment (jimaek) [inline files/astro/update.json]: There are no such files here https://github.com/Ting-y/Astro/tree/0.2.0

**Comment by Ting-y** [comment]
@jimaek Thanks for heads-up, i have changed my ting-y/Astro repository and add those files in it

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by jimaek** [comment]
Thank you

**Pull request closed without merging**